### ISSUE TITLE
Causing issue with react-toolbox component

### ISSUE THREAD

**Issue by ideletemyself** [csstools/postcss-font-magician#40]
Just a heads up, not sure if this is a problem for your end or not but when I try using this plugin with a basic react-toolbox project based off their example it seems to mess with the entire projects font-family and some of it's animation effects as well(ie -the sidebar in this example on their [playground](http://react-toolbox.com/#/components/layout)) even if I try specifically applying the font-family to say a class or id or even just a `<p>` tag. If I make a change it will effect the menu font-icon for say the top navigation "AppBar" ... So, I don't know if I'm doing something wrong but I don't think that I am, your plugin seems pretty straightforward.

Guess I should also mention that I can change fonts & font-icons np without conflict by following the instructions from Google fonts, so I don't think it's an issue with how I'm using react-toolbox. Also, I'm using Webpack but it hasn't got a lot going on since it's just a starter project based off the react-toolbox example and besides this plugin I'm only using cssnext as my only other PostCSS plugin.

You've got a handy cool plugin, love to get it working but no biggie.. Thanks!

**Comment by rajdee** [comment]
Hi @ideletemyself 
Thank you for your review, but I didn't understand what exactly your problem is? :) The animation doesn't work in react-toolbox or something else?

**Comment by ideletemyself** [comment]
> Hi @ideletemyself
Thank you for your review, but I didn't understand what exactly your problem is? :) The animation doesn't work in react-toolbox or something else?

Well, it's not just the animation that gets affected. It will also mess with whatever font-icons are being used. Admittedly, I don't have a lot of xp with React or react-toolbox for that matter. But since as I stated before, I can get different fonts, font-icons and whatever else working with no issues but then I use this plugin and things get messed up. So I'm assuming it's due to how this plugin either pulls the fonts or presents them and most likely isn't due to react-toolbox itself.

It not a big issue I just stopped using this plugin but as I said before it is pretty great, makes life easy when it works so would be cool to have it working with no issues with react-toolbox.

I could provide a couple small gif animations that show exactly what I'm talking about if you think that would help. Just didn't want to go to that trouble if I didn't have to lol. Thanks!
